velox/functions/prestosql/Arithmetic.h

```diff
@@ -495,5 +495,17 @@ struct TruncateFunction {
   }
 };
 
+template <typename T>
```

Comment:
Like @mbasmanova suggested in https://github.com/facebookincubator/velox/pull/4661, lets move the probability functions into their own file. Please resolve this after PR# 4661

Reply:
done